### ISSUE TITLE
[IOS] requestPayment does not throw error if occure

### ISSUE THREAD

**Issue by kefahB** [triniwiz/nativescript-plugins#134]
Hi @triniwiz 

In IOS standard implementation, requestPayment will return always success (via the listener) even if createPaymentSession dont return a payment intent at all!

I've tried to reject a promise into capturePayment method and it return always success.

**Comment by kefahB** [comment]
According to [the Stripe docs](https://stripe.dev/stripe-ios/docs/Protocols/STPPaymentContextDelegate.html#/c:@M@Stripe@objc(pl)STPPaymentContextDelegate(im)paymentContext:didCreatePaymentResult:completion:) we should handle 3 type of responses : 
- STPPaymentStatusSuccess
- STPPaymentStatusError
- STPPaymentStatusUserCancellation

For instance only the success is handled by this plugin, I suggest to update this part for IOS as below if you agree ?


```ts
StripePaymentDelegate.prototype.paymentContextDidCreatePaymentResultCompletion = function (paymentContext, paymentResult, completion) {
        StripeStandardConfig.shared.backendAPI
            .capturePayment(paymentResult.paymentMethod.stripeId, paymentContext.paymentAmount, createShippingMethod(paymentContext), createAddress(paymentContext.shippingAddress))
            .then(function (value) {
                if(!value._native.lastPaymentError || value._native.lastPaymentError == "undefined") {
                    completion(STPPaymentStatus.Success, null);
                    return
                }
                completion(STPPaymentStatus.UserCancellation, null);
            })
            .catch(function (e) {
            console.error("ERROR >> : paymentContextDidCreatePaymentResultCompletion : ", e);
            completion(STPPaymentStatus.Error, createError('PaymentError', 100, e));
        });
    };

```

**Comment by kefahB** [comment]
fixed with #141
